webapp/utils/utils.jsx

```diff
@@ -678,7 +678,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .emoji-picker__preview', 'border-top-color:' + changeOpacity(theme.centerChannelColor, 0.2));
         changeCss('.app__body .emoji-picker__category, .app__body .emoji-picker__category:focus, .app__body .emoji-picker__category:hover', 'color:' + changeOpacity(theme.centerChannelColor, 0.3));
         changeCss('.app__body .emoji-picker__category--selected, .app__body .emoji-picker__category--selected:focus, .app__body .emoji-picker__category--selected:hover', 'color:' + theme.centerChannelColor);
-        changeCss('.app__body .emoji-picker__item:hover', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
+        changeCss('.app__body .emoji-picker-items__container >span:hover', 'background-color:' + changeOpacity(theme.centerChannelColor, 0.8));
```

Comment:
Can you assign a class name to that span instead of using a child selector here? I'd prefer to be more descriptive about what we're setting the colour of here